service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsController.java

```diff
@@ -71,7 +71,7 @@ private NoticeOfProceedingsController(ObjectMapper mapper,
     }
 
     @PostMapping("/about-to-start")
-    public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody CallbackRequest callbackrequest) {
+    public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody CallbackRequest callbackrequest) {
```

Comment:
hmm good spot, try not to refactor things in files you aren't touching. Inflates the PR 🤷‍♂

Reply:
yeah, bit of effort having to log it somewhere in tech debt + watching it never get done or requiring new ticket, but you're definitely right